### ISSUE TITLE
"Write down your seed" check can be cheated

### ISSUE THREAD

**Issue by leafcutterant** [darkwallet/darkwallet#123]
When you create a new wallet, the seed displayed can be selected and copied, then pasted on the next screen when it asks to verify we have it written down on paper. This defeats the point of making sure users have a back-up. Lazy users will note this and use it, providing greater possibility of losing their coins.

I suggest making the seed text area non-selectable.

**Comment by genjix** [comment]
I think it's better to let users have the ability to cut themselves. For instance, for entering a wallet password, the user should be able to leave the field empty (blank) but they should get a warning otherwise people will resort anyway to using 1 letter passwords.

**Comment by esclear** [comment]
Well, I have found it useful for testing purposes.

**Comment by leafcutterant** [comment]
@genjix I'm with you on warnings, but in my opinion the seed is something critical.
@da-kid No doubt, in alpha stage this comes handy.

**Comment by ghost** [comment]
I agree with @genjix. FWIW Electrum also allows cut and paste of the seed. Remember, users can reveal the seed in settings so there is no problem at a later point if they need to check.
